.travis.yml

```diff
@@ -9,7 +9,7 @@ before_script:
   - pushd kalibro_install
     # Remove bugged libzmq3 package, see https://github.com/travis-ci/travis-ci/issues/982 and https://github.com/travis-ci/travis-ci/issues/1715 for details
   - sudo apt-get remove libzmq3
-  - export KALIBRO_PROCESSOR_VERSION=v1.1.7
+  - export KALIBRO_PROCESSOR_VERSION=v1.2.1.alpha1
```

Comment:
This version has to be updated when we release a new KalibroProcessor version. Still, this should not block this PR because KalibroProcessor's `v1.2.1' will just update the KalibroClient version to `v4.0.0`. This version, in turn, is equal to the version `v4.0.0.alpha1` that KalibroProcessor's `v1.2.1.alpha1` uses.